tools/clang_tidy/lib/src/git_repo.dart

```diff
@@ -78,7 +78,7 @@ class GitRepo {
         'diff-tree',
         '--no-commit-id',
         '--name-only',
-        '--diff-filter=ACMRT', # Added, copied, modified, renamed, or type-changed.
```

Comment:
Doh sorry too many languages to keep track of 🤦‍♀️

Reply:
No worries! I missed it too...